go/ql/lib/semmle/go/frameworks/stdlib/Log.qll

```diff
@@ -8,7 +8,7 @@ import go
 module Log {
   private class LogFunction extends Function {
     LogFunction() {
-      exists(string fn | fn.matches(["Fatal%", "Panic%", "Print%"]) |
+      exists(string fn | fn.matches(["Fatal%", "Panic%", "Print%", "Output"]) |
```

Comment:
Note that, because of [this line](https://github.com/github/codeql/blob/93d9332ab4e2da51817900a2c41709b7e0e49653/go/ql/lib/semmle/go/frameworks/stdlib/Log.qll#L28-L29), any argument is considered a message component (and hence a sink to two queries), including `calldepth int`.

Reply:
You're right. I had to follow a slightly more complicated approach and introduce a new class instead of modifying the existing one to take this into account. See https://github.com/github/codeql/pull/15268/commits/cf945546b133077617465f3065486bc20f3fe7f7.

Reply:
One downside of the way you've done it is that there are now lots more instances of the `LogCall` class. An approach we've used elsewhere is to add a field containing the index of the first message component, and then `getAMessageComponent` by `getSyntacticArgument(i)` where `i >= ` this index. [This code](https://github.com/github/codeql/blob/93d9332ab4e2da51817900a2c41709b7e0e49653/go/ql/lib/semmle/go/frameworks/Spew.qll#L14-L44) is a good example.

Reply:
Good point. What about [now](https://github.com/github/codeql/pull/15268/commits/ea218293b98c897338e79c3ef2a32e96590029b8)?